### PR TITLE
Fb rfdap 885 format table values

### PR DESCRIPTION
**Pull request by lincrosenbach**

### CODE REVIEW

**Review by lvoliveira — APPROVED** [review]

File: src/ng-xform/select-field/select-field.component.spec.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { ComponentFixture, fakeAsync, TestBed, tick } from '@angular/core/testing';
 import { FormControl } from '@angular/forms';
 import { By } from '@angular/platform-browser';
-import { of } from 'rxjs';
+import { of, Observable } from 'rxjs';
```
Comment (edetec) [inline src/ng-xform/select-field/select-field.component.spec.ts]: Where are you using it?

File: src/ng-xform/select-field/select-field.component.spec.ts [hunk 2]
```diff
@@ -43,6 +44,28 @@ describe('SelectFieldComponent', () => {
     expect(selectComponet).toBeTruthy();
   });
 
+  it('should render SelectField with searchHandler', () => {
```
Comment (igortg) [inline src/ng-xform/select-field/select-field.component.spec.ts:47]: Nice

**Review by igortg — COMMENTED** [review]
Can you add `formatOptions` on the `MeasureField` sample of README?

**Review by igortg — APPROVED** [review]

**Comment by lincrosenbach** [comment]
Added to README example of MeasureField with formatOptions

**Review by edetec — CHANGES_REQUESTED** [review]

File: src/ng-xform/select-field/select-field.component.spec.ts (hunk 1 above)
Comment (lincrosenbach) [reply src/ng-xform/select-field/select-field.component.spec.ts]: Removed.

**Review by edetec — APPROVED** [review]

**Pull request merged**